examples/mnist/.tensorleap/dataset.py

```diff
@@ -196,7 +200,7 @@ def metadata_most_similar_class_not_gt_diff(idx: int, subset: Union[SubsetRespon
                                 subset='images',
                                 ground_truth_type=DatasetOutputType.Classes,
                                 name='classes',
-                                labels=labels,
+                                labels=LABELS,
                                 masked_input=None)
 
 # Set Metadata
```

Comment:
Consider changing to something like `Bind Metadata Functions` . Or remove this comment and keep one comments for the binding section.